.github/workflows/codestyle.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       
       - name: Show entire output pull request
         if: github.event_name == 'pull_request_target'
-        run: ruff check --output-format=concise $(git --no-pager diff --name-only origin/master HEAD --) | tee ruff_full.txt
+        run: ruff check --output-format=concise $(git --no-pager diff --name-only origin/master HEAD -- | grep -v -E '\.(lock|yml|rst)$') | tee ruff_full.txt
```

Comment:
right now it checks- "all files changed in this PR except files ending in .lock, .rst and .yml". Is it better to change it "all python files changed in this PR?"